embedded/src/main/groovy/com/bmuschko/gradle/tomcat/embedded/BaseTomcatServerImpl.groovy

```diff
@@ -40,6 +40,11 @@ abstract class BaseTomcatServerImpl implements TomcatServer {
         context.loader.addRepository(resource.toURI().toURL().toString())
     }
 
+    @Override
+    void addResource(Resource resource) {
+        // TODO: how to implement this here for Tomcat 7x?
```

Comment:
However I haven't found out how to do it on Tomcat7.

I'm not sure if it's worth finding a solution for Tomcat7; maybe Tomcat7 should be removed anyway and instead support for Tomcat 10 be implemented as a replacement? #204